lib/internal/process/next_tick.js

```diff
@@ -19,6 +19,69 @@ function setupNextTick() {
   const { kInit, kDestroy, kAsyncIdCounter } = async_wrap.constants;
   const { async_id_symbol, trigger_async_id_symbol } = async_wrap;
 
+  class SmallQueue {
+    constructor() {
+      this.list = [];
+      this.top = 0;
+      this.btm = 0;
+    }
+
+    push(tick) {
+      if (this.top === this.list.length) {
+        this.list.push(tick);
+        this.top++;
+      } else {
+        this.list[this.top++] = tick;
+      }
+    }
+
+    shift() {
+      if (this.btm < this.top) {
+        var next = this.list[this.btm];
```

Comment:
`const`?

Reply:
I think using `const` within an if used to cause some issues with V8. I do not know if this was fixed recently or not. cc @bmeurer